This static assert does not take into account that the package can be build
for both 64 and 32bit.
--- a/vendor/ffmpeg-sys-next-9.0.0/hwcontext_stubs/vulkan/vulkan.h
+++ b/vendor/ffmpeg-sys-next-9.0.0/hwcontext_stubs/vulkan/vulkan.h
@@ -61,7 +61,5 @@
     void           *pNext;
     VkBool32        features[55];
 } VkPhysicalDeviceFeatures2;
-_Static_assert(sizeof(VkPhysicalDeviceFeatures2) == 240,
-               "VkPhysicalDeviceFeatures2 must be 240 bytes (Vulkan 1.1 frozen ABI)");

 #endif
